R/scale-product.R

```diff
@@ -31,7 +31,12 @@ product_labels <- function() {
 
 is.waive <- getFromNamespace("is.waive", "ggplot2")
 
-
+## Fix for ggplot_3.3
+if (utils::packageVersion("ggplot2") >= "3.2.1.9000") {
```

Comment:
This is outside of a function, which means this is determined on installation. Though I guess this works in most cases, there's a chance the user installs ggmisaic when their ggplot2 is version 3.2.1 and then update ggplot2 to version >=3.3.0.

What about setting `ggplot2::waiver()` as the default and replace it with `"none"` in `scale_(x|y)_productlist()` only when the version of ggplot2 is <= 3.2.1?